docs/blog/newsletter-june-2017

```diff
@@ -0,0 +1,110 @@
+#######################################
+
+`Write the Docs Portland <http://www.writethedocs.org/conf/na/2017/>`_ was another sold-out success! Many thanks to everyone who attended, presented, hiked, shared, discussed, and made it another great event. Videos of the presentations are on the way--watch your email and the WTD Slack channels for an announcement when they're posted to YouTube.
+
+Now, it's time to gear up for `Write the Docs Prague <http://www.writethedocs.org/conf/eu/2017/>`_ in September!
+
+This month's newsletter is the first one that relies on your `Reacji Channeler <https://reacji-channeler.builtbyslack.com>`_suggestions! As a reminder, you can tag messages as suggestions for the newsletter with this emoji:
+
+.. image:: news.png
+    :width: 128px
+    :align: left
+    :height: 128px
+    :alt: emoji for tagging newsletter suggestions
+
+And thank you for all of your suggestions!
+
+------------------------------------------
+
+Writing for Write the Docs on Writing Day
+#########################################
+
+In Portland this year, for the first time ever, we ran a WTD session on writing day! In addition to learning a *lot* about how to run writing day sessions :smiling_imp:, we worked with around a dozen helpful attendees to improve the WTD website in `various ways <https://github.com/writethedocs/www/issues?utf8=%E2%9C%93&q=label%3Awritingday%20>`_:
+
+* Lots of progress toward a unified talk video archive
+* Cleaned up many broken links and build warnings
+* Improved the information on how to build the site on Windows
+* Added links to write-ups of the 2016 Portland conference
+
+Thanks to everyone who helped out--see you at the write table in Prague or back in Portland next year!
+
+------------------------------------------
+
+Getting a Grip on Static Site Generators
+########################################
+
+Static site generators have been a popular topic for some time, but there are a lot of moving parts that can be intimidating for new users. One Slack participant wondered how static site tools like Sphinx, Asciidoctor, restructured text (rST), Jekyll, and Markdown fit together--or don't. Here's the insight from our community:
+
+* Sphinx, Jekyll, and Asciidoctor are parsing engines; rST, Markdown, and AsciiDoc are source file formats; and Python and Ruby are programming languages. So you can write in rST to publish with Sphinx, which is written in Python. Or, you can write in Markdown to publish in Jekyll, which is written in Ruby.
+* In general, Sphinx parses rST, and Jekyll parses Markdown. However, some flavors of Sphinx parse Markdown too, like `Read the Docs <https://docs.readthedocs.io/en/latest/>`_.
+* There are also plugins out there to translate Markdown to rST, and to make rST work with Jekyll. An example is `recommonmark <http://recommonmark.readthedocs.io/en/latest/>`_.
+* You write in Markdown to publish with Jekyll, but Jekyll actually publishes the static site in HTML.
+
```

Comment:
I might be able to cover this in a nicer way -- but I'd probably end up with a blog post at the end of it, so maybe I should write a blog post, but I likely won't have time to do it before we send the newsletter :)